testing/php7-pecl-seaslog/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: viest <dev@service.viest.me>
+# Maintainer: Chitao.Gao <neeke@php.net>
+pkgname="php7-pecl-seaslog"
+_pkgreal=SeasLog
+pkgver="2.0.2"
+pkgrel=0
+pkgdesc="An effective,fast,stable log extension for PHP."
+url="https://pecl.php.net/package/SeasLog"
+arch="all"
+license="PHP 3.01"
+depends="php7-common"
+makedepends="php7-dev"
+source="$pkgname-$pkgver.tar.gz::https://pecl.php.net/get/$_pkgreal-$pkgver.tgz"
+builddir="$srcdir"/$_pkgreal-$pkgver
+
+build() {
+    phpize7
+    ./configure \
+        --prefix=/usr \
+        --with-php-config=/usr/bin/php-config7
+    make
+}
+
+check() {
+    make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 test
+}
+
+package() {
+    make INSTALL_ROOT="$pkgdir/" install
+    install -d "$pkgdir"/etc/php7/conf.d
+    echo "extension=$_pkgreal.so" > "$pkgdir"/etc/php7/conf.d/$_pkgreal.ini
+}
+
+sha512sums="bc1dc5855461c510edfe2a4cb50cee18b98b8063bbd07957300842073530cd7deae54e3c062ca175e11df9108dde4341cc75ba78376a3b95516509a9c2a0beae  php7-pecl-seaslog-2.0.2.tar.gz"
```

Comment:
nit: no new line, use `abuild checksum` to fix or just manually

Reply:
I think the newline is still missing, makes for kind of ugly diffs :/